### PR TITLE
Adding a redirect for web/experimental/

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: web/experimental/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+        <meta http-equiv="refresh" content="0; url=embed-new.html" />
+    </head>
+    <body>
+        Redirecting to <a href="embed-new.html">embed-new.html</a>
```
Comment (RedBrogdon) [inline web/experimental/index.html]: I'd probably point people to `new_embeddings_demo.html` instead.

File: web/experimental/index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    </body>
+</html>
```
Comment (RedBrogdon) [inline web/experimental/index.html]: newline?

File: web/experimental/index.html (hunk 1 above)
Comment (domesticmouse) [reply web/experimental/index.html]: Done,

File: web/experimental/index.html (hunk 2 above)
Comment (domesticmouse) [reply web/experimental/index.html]: and done.

**Comment by domesticmouse** [comment]
I like having sane `index.html` files in directories because I traverse websites by editing URLs. I know, I'm OG.

**Pull request merged**